### ISSUE TITLE
Domain management

### ISSUE THREAD

**Issue by arnaudbenard** [opencollective/opencollective-app#59]
We currently have 3 separate websites:
a) Website (`website` splash page) - http://opencollective.com/
b) Mobile app (`mobileapp`) -  http://opencollective-staging-app.herokuapp.com/
c) Web app (web version of `mobileapp`) - http://opencollective-staging-app.herokuapp.com/public/groups/12

Right now I don't have the perfect solution to unite all those websites but here are some thoughts I had.

1) The website a) looks pretty much like the web app c). We could merge the `mobileapp` repo with `website`
2) We could nest all the pages of b) inside `/app` and make `app.opencollective.com` point to `opencollective-prod-app.com/app/`.

Basically the logic is to merge everything inside `mobileapp` and to make all the urls that are not inside `opencollective-prod-app.com/app/` public. That way we can have `opencollective.com/1/yeoman` (check comment https://github.com/OpenCollective/opencollective-mobileapp/pull/57#issuecomment-167069797) and `opencollective.com/about`.

I would love some feedback :)
